{"uuid": "ed0ff2d1-42d9-4963-9ba8-2cab699da5c2", "children": ["395434be-df31-4262-a911-1725676689ae"], "befores": [{"name": "driver", "status": "passed", "start": 1691765183265, "stop": 1691765185022}], "afters": [{"name": "driver::0", "status": "broken", "statusDetails": {"message": "AttributeError: 'Function' object has no attribute 'rep_call'\n", "trace": "  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 917, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\code\\test\\pythonCode\\pythonProject\\ui_pytest_framework\\conftest.py\", line 15, in driver\n    if request.node.rep_call.failed:\n       ^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1691765188598, "stop": 1691765188598}], "start": 1691765183265, "stop": 1691765188598}